integration/integration_test.go

```diff
@@ -522,6 +525,75 @@ func TestSeccompadvisor(t *testing.T) {
 	runCommands(commands, t)
 }
 
+func TestAuditSeccomp(t *testing.T) {
+	ns := generateTestNamespaceName("test-audit-seccomp")
+
+	t.Parallel()
+
+	commands := []*command{
+		createTestNamespaceCommand(ns),
+		{
+			name: "Create SeccompProfile",
+			cmd: fmt.Sprintf(`
+				kubectl apply -f - <<EOF
+apiVersion: security-profiles-operator.x-k8s.io/v1beta1
+kind: SeccompProfile
+metadata:
+  name: log
+  namespace: %s
+  annotations:
+    description: "Log some syscalls"
+spec:
+  defaultAction: SCMP_ACT_ALLOW
+  architectures:
+  - SCMP_ARCH_X86_64
+  syscalls:
+  - action: SCMP_ACT_KILL
+    names:
+    - unshare
+  - action: SCMP_ACT_LOG
+    names:
+    - mkdir
+EOF
+			`, ns),
+			expectedRegexp: "seccompprofile.security-profiles-operator.x-k8s.io/log created",
+		},
+		{
+			name: "Run test pod",
+			cmd: fmt.Sprintf(`
+				kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+  namespace: %s
+spec:
+  securityContext:
+    seccompProfile:
+      type: Localhost
+      localhostProfile: operator/default/log.json
```

Comment:
The Github action still uses Minikube with `kubernetes version: 'v1.18.2'`. We need a more recent version to use the `seccompProfile` field in the `securityContext`

Reply:
Hum...
Maybe we will need to enable back the flag which was deleted in 9cc56397e957341989040a95508d810b845b7b7a.

Reply:
I think it's possible to keep that test running, we should just use Kubernetes v1.19. I'm testing this in https://github.com/kinvolk/inspektor-gadget/pull/564.